File: src/main/java/com/fatec/destino/dto/pacote/local/IbgeEstadoDTO.java
package com.fatec.destino.dto.pacote.local;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public record IbgeEstadoDTO(Long id, String sigla, String nome, IbgeRegiaoDTO regiao) {}

File: src/main/kotlin/com/fatec/destino/dto/local/IbgeEstadoDTO.kt
package com.fatec.destino.dto.local

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class IbgeEstadoDTO(
    var id: Long,
    var sigla: String,
    var nome: String,
    var regiao: IbgeRegiaoDTO
)
